website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx

```diff
@@ -106,9 +106,9 @@ Review a list of tags for all resources across the organization. When you delete
 <!-- END: TFC:only name:pnp-callout -->
 
 
-All users in an organization can access the **Teams** page, which displays a list of [teams][] within the organization. This excludes [secret teams](/terraform/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
+All users in an organization can access the **Teams** page, which displays a list of [teams][] within the organization. Owners and users with [Include Secret Teams](/terraform/cloud-docs/users-teams-organizations/permissions#include-secret-teams) can view all [secret teams](/terraform/cloud-docs/users-teams-organizations/teams#team-visibility) within the organization. You can also view team membership and manage team API tokens.
 
-Organization owners can also create and delete teams, and manage team API tokens. Both owners and users with [Manage Membership](/terraform/cloud-docs/users-teams-organizations/permissions#manage-membership) permissions can manage the membership of teams. Remember that users must accept an invitation to the organization before you can add them to teams.
+The ability to create and delete teams, and manage team API tokens is restricted to organization owners and users with [Manage Teams](/terraform/cloud-docs/users-teams-organizations/permissions#manage-teams) permission. Both owners and users with [Manage Membership](/terraform/cloud-docs/users-teams-organizations/permissions#manage-membership) permissions can manage the membership of teams. Remember that users must accept an invitation to the organization before you can add them to teams.
```

Comment:
Non-blocking Nitpick (similar to above!): I feel like this reads nicer with "they" here to avoid switching points of view:
```suggestion
The ability to create and delete teams, and manage team API tokens is restricted to organization owners and users with [Manage Teams](/terraform/cloud-docs/users-teams-organizations/permissions#manage-teams) permission. Both owners and users with [Manage Membership](/terraform/cloud-docs/users-teams-organizations/permissions#manage-membership) permissions can manage the membership of teams. Remember that users must accept an invitation to the organization before they can be added to teams.
```